pymc/aesaraf.py

```diff
@@ -535,55 +535,155 @@ def make_shared_replacements(point, vars, model):
 
 def join_nonshared_inputs(
     point: Dict[str, np.ndarray],
-    xs: List[TensorVariable],
-    vars: List[TensorVariable],
-    shared,
-    make_shared: bool = False,
-):
+    outputs: List[TensorVariable],
+    inputs: List[TensorVariable],
+    shared_inputs: Optional[Dict[TensorVariable, TensorSharedVariable]] = None,
+    make_inputs_shared: bool = False,
+) -> Tuple[List[TensorVariable], TensorVariable]:
     """
-    Takes a list of Aesara Variables and joins their non shared inputs into a single input.
+    Create new outputs and input TensorVariables where the non-shared inputs are joined
+    in a single raveled vector input.
 
     Parameters
     ----------
-    point: a sample point
-    xs: list of Aesara tensors
-    vars: list of variables to join
+    point: dict of {str : array_like}
```

Comment:
```suggestion
    point : dict of {str : array_like}
```

All parameters **and** return values should have a space before and after the colon. You will then see in the preview that known/understood types are rendered as hyperlinks

Reply:
Subtle. Thanks for pointing out. Will make change